server/handle_common.go

```diff
@@ -8,8 +8,8 @@ package server
 import "github.com/gin-gonic/gin"
 
 type ErrorResult struct {
-	ErrCode int    `json:"err_code,omitempty"`
-	ErrMsg  string `json:"err_msg,omitempty"`
+	ErrCode int    `json:"err_code,omitempty" binding:"required"`
```

Comment:
actually the error code here is not needed.. since the actual error code is sent in the HTTP code. If we can control the error code in the HTTP header, that is enough. 
If needed, we can have some internal codes or strings (enums) to define what each error means. Then the client can differentiate the type of error. Similar to what we have in xpanse.

Reply:
I deleted the error code in the struct ErrorResult